### PR TITLE
Initial Task Hierarchy Revision & Auto Update Readme

### PR DESCRIPTION
**Pull request by garyhlai**
- Add a script to auto update the corresponding task line in README after modifying the task file
- Add two columns in tasks/README.md
- Revise two task labels according to the new task hierarchy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,8 @@ We would appreciate any external contributions! 🙏
     * If you're building your tasks based existing datasets and their crowdsourcing templates, see these [guidelines](doc/crowdsourcing.md). 
  * Add your task to [our list of tasks](tasks/README.md).
  * To make sure that your addition is formatted correctly, run the tests: `> python src/test_all.py`
-  
+ * To automatically update README after modifying "Categories", "Domains", "Input_language" or "Output_language", run : `> python src/auto_update_readme.py`
```
Comment (danyaljj) [inline README.md]: I would prefer if we drop this. Most users don't need to worry about this.

**Comment by danyaljj** [comment]
One comment; otherwise, looks good.

File: README.md (hunk 1 above)
Comment (garyhlai) [reply README.md]: Done!

**Comment by garyhlai** [comment]
@danyaljj Btw should I change the file names as well? e.g. `task001_quoref_question_generation.json` --> `task001_quoref_contextual_question_generation.json` ?

**Comment by danyaljj** [comment]
Good question! For now, let's not touch the file names.

**Pull request merged**